bitnami/rabbitmq-cluster-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,55 @@
+{{/*
+Return the proper RabbitMQ Cluster Operator image name
+*/}}
+{{- define "rmqco.operator.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper image name (for the init container volume-permissions image)
+*/}}
+{{- define "rmqco.rabbitmq.image" -}}
+{{- include "common.images.image" ( dict "imageRoot" .Values.rabbitmqImage "global" .Values.global ) -}}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "rmqco.imagePullSecrets" -}}
+{{- include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.rabbitmqImage) "global" .Values.global) -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "rmqco.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{ default (printf "%s" (include "common.names.fullname" .)) .Values.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names (deprecated: use common.images.renderPullSecrets instead)
+{{ include "common.images.pullSecrets" ( dict "images" (list path.to.the.image1, path.to.the.image2) "global" .Values.global) }}
+*/}}
+{{- define "rmqco.defaultPullSecrets" -}}
+  {{- $pullSecrets := list }}
+  {{- if .global }}
+    {{- range .global.imagePullSecrets -}}
+      {{- $pullSecrets = append $pullSecrets . -}}
+    {{- end -}}
+  {{- end -}}
+
+  {{- range .images -}}
+    {{- range .pullSecrets -}}
+      {{- $pullSecrets = append $pullSecrets . -}}
+    {{- end -}}
+  {{- end -}}
+{{- if (not (empty $pullSecrets)) }}
+{{- range $pullSecrets }}
+- name: {{ . }}
+{{- end }}
+{{- end }}
+{{- end -}}
```

Comment:
You're using a deprecated way to define the pull secrets macro.

Reply:
Why do they need to be defined by default?

Reply:
Because then the default image we set for the RabbitmqCluster instances would not be accessible. The user would need to re-set every time when they deploy RabbitmqCluster objects and the upstream examples would not work by default.

Reply:
So.. It's because the upstream examples are using a private image, am i right?